pkgs/development/interpreters/octave/build-octave-package.nix

```diff
@@ -123,10 +123,6 @@ stdenv.mkDerivation (
       else
         preBuild;
 
-    # Add Octave's shared objects to ld library path for some packages which
-    # need to link their C/C++/Fortran code against Octave's internals.
-    LD_LIBRARY_PATH = lib.makeLibraryPath nativeBuildInputs';
```

Comment:
I'm confused. As a start can we squash this addition and then removal and hence the whole PR to 1 commit?

Reply:
Yes, that was the plan. The `LD_LIBRARY_PATH` line commit will be removed.